### PR TITLE
[WSTEAM1-300] Expand download functionality to all podcast services

### PR DESCRIPTION
**Pull request by mfonofm**
Resolves JIRA [300]

Overall changes
======
Expands the download functionality to all podcast services (previously only on Burmese service)

Code changes
======
- Update tests and snapshots
- Expand download functionality to all Podcast services
- Add download translations to `PodcastExternalLinks`
- Add download translations to config

Testing
======
1. Pull down this PR
2. Go to a podcast page: http://localhost:7080/arabic/podcasts/p02pc9qc or http://localhost:7080/hausa/podcasts/p08mlgcb
3. Observe that these services now have a 'download' button within the podcast external links section

Helpful Links
======
[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: src/app/legacy/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap [hunk 1]
```diff
@@ -226,19 +226,19 @@ exports[`PodcastExternalLinks Should render external links 1`] = `
         dir="ltr"
       >
         <a
-          aria-labelledby="externalLinkId-Download"
+          aria-labelledby="externalLinkId-Загрузить эпизод"
```
Comment (amoore108) [inline src/app/legacy/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap]: May need to apply some sanitisation to the IDs here to remove whitespace. 

There is a utility https://github.com/bbc/simorgh/blob/cc67358813074ae264e6cd9e63aba3818eb3bb3d/src/app/lib/utilities/idSanitiser/index.js that could do the job.

File: src/app/lib/config/services/indonesia.ts [hunk 2]
```diff
@@ -239,6 +239,7 @@ export const service: DefaultServiceConfig = {
         duration: 'Durasi',
         recentEpisodes: 'Siaran sebelumnya',
         podcastExternalLinks: 'Podcast ini juga tersedia di',
+        download: ' 	Unduh episode',
```
Comment (amoore108) [inline src/app/lib/config/services/indonesia.ts]: Some whitespace here, not sure if needed.

File: src/app/routes/onDemandAudio/tempData/podcastExternalLinks/index.js [hunk 3]
```diff
@@ -48,8 +47,7 @@ export const getPodcastExternalLinks = async (
     if (!brandPid) return [];
     let downloadLink = [];
     const links = pathOr([], ['default', variant, brandPid], linkData);
-    // Burmese podcast experiment
-    if (service === 'burmese' && brandPid === 'p02pc9lh') {
+    if (service) {
```
Comment (amoore108) [inline src/app/routes/onDemandAudio/tempData/podcastExternalLinks/index.js]: Think we may be able to remove this condition as this function should only run on services that have podcast pages.

**Comment by amoore108** [comment]
Just a couple of queries, but looks good! 👏

**Review by karinathomasbbc — APPROVED** [review]
🎙️

File: src/app/routes/onDemandAudio/tempData/podcastExternalLinks/index.js (hunk 3 above)
Comment (mfonofm) [reply src/app/routes/onDemandAudio/tempData/podcastExternalLinks/index.js]: Good spot!

File: src/app/legacy/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap (hunk 1 above)
Comment (mfonofm) [reply src/app/legacy/containers/PodcastExternalLinks/__snapshots__/index.test.jsx.snap]: 👍

**Review by amoore108 — APPROVED** [review]

**Pull request merged**